docs-2.0/3.ngql-guide/15.text-based-index-statements/4.search-with-fulltext-index.md

```diff
@@ -0,0 +1,59 @@
+# Full-text search
+
+```ngql
+LOOKUP ON {<tag> | <edge_type>} WHERE <expression> [YIELD <return_list>]
+
+<expression>
+    PREFIX | WILDCARD | REGEXP |FUZZY
+
+<return_list>
+    <col_name> [AS <col_alias>] [, <col_name> [AS <col_alias>] ...]
+```
+
+Use the `LOOKUP ON` statement to do full-text search. The search string is specified in the `WHERE` clause.
+
+## Natural language full-text search
+
+A natural language search interprets the search string as a phrase in natural human language. The search is case-insensitive.
+
+## Examples
+
+```ngql
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE WILDCARD(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE REGEXP(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a");
```

Comment:
examples are not good.
It looks like grammar, not examples.
You shall explain the parameters in    (1,2,3,4,5)